.eslintrc.js

```diff
@@ -2,7 +2,17 @@ module.exports = {
   extends: 'airbnb',
   rules: {
     'no-console': 'off',
-    'no-underscore-dangle': ['error', { allow: ['_id'] }],
-    'no-plusplus': 'off'
+    'no-underscore-dangle': ['error', { allow: ['_id', '__REDUX_DEVTOOLS_EXTENSION__'] }],
+    'no-plusplus': 'off',
+    camelcase: 'off',
+    'no-mixed-operators': 'off',
+    'react/jsx-filename-extension': 'off',
```

Comment:
We should discuss eslint rules in our upcoming meeting so that everyone is using the same ones.